src/Tribe/Views/V2/View.php

```diff
@@ -1405,6 +1406,9 @@ static function () {
 			}
 		);
 
+		/** @var Rest_Endpoint $endpoint */
+		$endpoint      = tribe( Rest_Endpoint::class );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 6 spaces

<sub>hash: d0b95eb1e4d2fc897af4254df8f1eaf6</sub>